site/content/posts/2023-08-08-v0.25.0-impersonation-proxy-with-external-certs.md

```diff
@@ -0,0 +1,318 @@
+---
+title: "Pinniped v0.25.0: With External Certificate Management for the Impersonation Proxy and more"
+slug: v0-25-0-external-cert-mgmt-impersonation-proxy
+date: 2023-08-08
+author: Joshua T. Casey
+image: https://images.unsplash.com/photo-1618075254460-429d47b887c7?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2148&q=80
+excerpt: "With v0.25.0 you get external certificate management for the impersonation proxy, easier scheduling of the kube-cert-agent, and more"
+tags: ['Joshua T. Casey','Ryan Richard', 'Benjamin Petersen', 'release']
```

Comment:
As far as I can tell, we don't use tags in the generated HTML 🤔

Reply:
Odd. You are correct. Not sure what we should do about that or our SEO tags now.

Reply:
The tags are listed here: https://pinniped.dev/tags/

Reply:
https://gohugo.io/content-management/taxonomies/

Reply:
Woof, page looks terrible.